docs/tutorial/workers.md

```diff
@@ -126,95 +123,132 @@ From our previous steps, we now have:
+    ```python hl_lines="21-25" title="repo_info.py"
+    import httpx
+    from prefect import flow
+    from prefect.deployments import DeploymentImage
 
-Prefect will now build a custom Docker image containing your workflow code that the worker can use to dynamically spawn Docker containers whenever this workflow needs to run. 
 
-### Modify the deployment
+    @flow(log_prints=True)
+    def get_repo_info(repo_name: str = "PrefectHQ/prefect"):
+        url = f"https://api.github.com/repos/{repo_name}"
+        response = httpx.get(url)
+        response.raise_for_status()
+        repo = response.json()
+        print(f"{repo_name} repository statistics 🤓:")
+        print(f"Stars 🌠 : {repo['stargazers_count']}")
+        print(f"Forks 🍴 : {repo['forks_count']}")
 
-If you selected `y` on the last prompt to save configuration, you should see a new `prefect.yaml` file appear. This file will allow you to easily modify and define multiple deployments for this repo.
-
-The [`prefect.yaml`](/guides/prefect-deploy/#managing-deployments) file not only holds settings for various deployments but can also contain instructions that help set up the execution environment for your flow runs. In the context of this tutorial, we employ a [build action](/guides/prefect-deploy/#the-build-action) to create a Docker image that the worker will use when running your flow.
-
-Upon examining the auto-generated `prefect.yaml`, you'll notice that the parameters for your deployment mirror the values you provided to the deployment creation wizard:
-
-```yaml title="prefect.yaml" hl_lines="2 10 20"
-build:
-- prefect_docker.deployments.steps.build_docker_image:
-    requires: prefect-docker>=0.3.1
-    id: build-image
-    dockerfile: auto
-    image_name: docker-user/deployment-image
-    tag: latest
-
-deployments:
-- name: my-deployment
-  version: null
-  tags: []
-  description: null
-  entrypoint: my_flow.py:get_repo_info
-  parameters: {}
-  work_pool:
-    name: my-docker-pool
-    work_queue_name: null
-    job_variables:
-      image: '{{ build-image.image }}' ## Resultant image from the build action
-  schedule: null
-```
 
-It's worth noting that the `prefect.yaml` supports [referencing dynamic values](/guides/prefect-deploy/#templating-options). You can see that our deployment references the Docker image produced from the build action above.
+    if __name__ == "__main__":
+        get_repo_info.deploy(
+            name="my-first-deployment", 
+            work_pool_name="my-docker-pool", 
+            image=DeploymentImage(
+                name="my-first-deployment-image",
+                tag="tutorial",
+                dockerfile="Dockerfile"
+            ),
+            skip_push=True
+        )
+    ```
+
+### Modify the deployment
```

Comment:
I realize this document isn't consistent in using either the infinitive verb for ("modify") or the present participle form ("modifying"). I think it would look nice for the headings to be consistent there

Reply:
I've updated the header to all use the infinitive form.